doc/sphinxman/source/optking.rst

```diff
@@ -262,45 +258,219 @@ For bends, the corresponding keyword is "frozen_bend".
+    | theta_A | angle    | A1-A0-B0    | A has > 1 atom                  |
+    +---------+----------+-------------+---------------------------------+
+    | theta_B | angle    | A0-B0-B1    | B has > 1 atom                  |
+    +---------+----------+-------------+---------------------------------+
+    | tau     | dihedral | A1-A0-B0-B1 | A and B have > 1 atom           |
+    +---------+----------+-------------+---------------------------------+
+    | phi_A   | dihedral | A2-A1-A0-B0 | A has > 2 atoms. Is not linear  |
+    +---------+----------+-------------+---------------------------------+
+    | phi_B   | dihedral | A0-B0-B1-B2 | B has > 2 atoms. Is not linear  |
+    +---------+----------+-------------+---------------------------------+
+
+* A constrained optimization is performed where the orientation of the two fragments is fixed but
+  the distance between the fragments and all intrafragment coordinates are allowed to relax.
+
+.. code-block:: none
+
+   memory 4GB 
+   molecule mol {
+     C       -1.258686      0.546935      0.436840
+     H       -0.683650      1.200389      1.102833
+     C       -0.699036     -0.349093     -0.396608
+     C       -2.693370      0.550414      0.355311
+     H       -3.336987      1.206824      0.952052
+     C       -3.159324     -0.343127     -0.536418
+     H       -4.199699     -0.558111     -0.805894
+     S       -1.883829     -1.212288     -1.301525
+     C        0.786082     -0.656530     -0.606057
+     H        1.387673     -0.016033      0.048976
+     H        1.054892     -0.465272     -1.651226
+     H        0.978834     -1.708370     -0.365860
+     --
+     C       -6.955593     -0.119764     -1.395442
+     C       -6.977905     -0.135060      1.376787
+     C       -7.111625      1.067403     -0.697024
+     C       -6.810717     -1.314577     -0.707746
+     C       -6.821873     -1.322226      0.678369
+     C       -7.122781      1.059754      0.689090
+     H       -7.226173      2.012097     -1.240759
+     H       -6.687348     -2.253224     -1.259958
+     H       -6.707325     -2.266920      1.222105
+     H       -7.246150      1.998400      1.241304
+     O       -6.944245     -0.111984     -2.805375
+     H       -7.058224      0.807436     -3.049180
+     C       -6.990227     -0.143507      2.907714
+     H       -8.018305     -0.274985      3.264065
+     H       -6.592753      0.807024      3.281508
+     H       -6.368443     -0.968607      3.273516
+     nocom
+     unit angstrom
+   }
+   
+   # Create a python dictionary and convert to string for pass through to optking
+   MTdimer = """{
+      "Natoms per frag": [12, 16],
+      "A Frag": 1,
+      "A Ref Atoms": [[1, 3, 4, 6, 8], [8], [11]],
+      "A Label": "methylthiophene",
+      "B Frag": 2,
+      "B Ref Atoms": [[13, 14, 15, 16, 17, 18], [13], [15]],
+      "B Label": "tyrosine",
+      "Frozen": ["theta_A", "theta_B", "tau", "phi_A", "phi_B"],
+   }"""
```

Comment:
It could be good to have a pytest test, maybe in test_addons.py, showing the passing through of optking opts in psiapi where you shouldn't have to resort to stringifying the dict.